test/system/scenarios/invalid-orders.js

```diff
@@ -0,0 +1,30 @@
+const Promish = require('promish');
+
+const run = (trader1, trader2, trader3) => {
+  console.log('****************************************************************');
+  console.log('** Scenario 13: Test Invalid Orders                         **');
+  console.log('****************************************************************');
+
+  return Promish.resolve()
+    .then(() => trader2.placeLimitOrder('B', -1, 0.11, { expectFail: true })) // negative qty
+    .then(() => trader3.placeLimitOrder('S', 1, -0.15, { expectFail: true })) // negative price
+    .then(() => trader1.placeLimitOrder('A', 1, 0.10, { expectFail: true })) // invalid side
+    .then(() => trader2.placeLimitOrder('', 1, 0.10, { expectFail: true })) // without side
```

Comment:
This line doesn't test limit order without a side, it tests a limit order with a side equal to empty string. Need to do something like this:
```
trader.createOrder({type: 'L', price: 0.1, quantity: 1}, {expectFail: true})
```